### PR TITLE
[vultr] Handle network request errors

### LINKED ISSUE #108: Vultr response when no server is up
I installed vultr command set and run the command vultr_list and got some rows with undefined as text.
better to show more informative. 

image

### PR DESCRIPTION
**Pull request by satyarohith**
Closes #108

### CODE REVIEW

File: vultr/packages/vultr/vultr_list.js [hunk 1]
```diff
@@ -67,39 +67,47 @@ async function _command(params, commandText, secrets = {}) {
     } catch {
       await install(['@vultr/vultr-node']);
       Vultr = require('@vultr/vultr-node');
-    }   
+    }
 
     const {server} = Vultr.initialize({apiKey: vultrApiKey});
 
     const data = await server.list();
```
Comment (v-shant) [inline vultr/packages/vultr/vultr_list.js:74]: getting this when I have no server. I think data is an array we may check the length.

image

Comment (v-shant) [reply vultr/packages/vultr/vultr_list.js:74]: change const result and const output to let.

**Review by v-shant — CHANGES_REQUESTED** [review]

File: vultr/packages/vultr/vultr_list.js (hunk 1 above)
Comment (satyarohith) [reply vultr/packages/vultr/vultr_list.js:74]: > getting this when I have no server. I think data is an array we may check the length.
> 
> image

Fixed it!

**Review by v-shant — APPROVED** [review]
changes LGTM. miner suggestion to change the use let instead of const eg in vultr_list.js `const result = [ ] `and `const output  = [ ]`

**Comment by satyarohith** [comment]
We're not reassigning anything to those variables. I'm curious to learn about the reason behind your suggestion. :)

**Comment by v-shant** [comment]
no particular reason just worried when we use const with an array and object and we have to change the values needs more care even let and const have the same scope and also for some developers like me it becomes a little bit confusing how it is working fine when we are declaring an array with const and push and pop still working. and also this code will have public access i thought it should be better not let them confuse with non-reinitializing and just pushing and poping.

But if it is fine for you we can keep const.

**Comment by satyarohith** [comment]
I can totally understand your concern, but to maintain consistency with other commands, let's keep it as it. :)

**Pull request merged**